cmd/snap-confine/snap-device-helper

```diff
@@ -15,9 +15,19 @@ MAJMIN="$4"
 [ -n "$DEVPATH" ] || { echo "no devpath given" >&2; exit 1; }
 [ -n "$MAJMIN" ] || { echo "no major/minor given" >&2; exit 0; }
 
-APPNAME="$( echo "$APPNAME" | tr '_' '.' )"
+NOSNAP="${APPNAME#snap_}"
+[ "$NOSNAP" != "$APPNAME" ] || { echo "malformed appname $APPNAME" >&2; exit 1; }
+
```

Comment:
can we also exit with an error if `NOSNAP` has more than two `_`s? (with a test for it :-) )